lib/sender.js

```diff
@@ -67,67 +68,91 @@ module.exports = {
 */
 function send(client) {
     return function send(message, options) {
-        return new Promise((resolve, reject) => {
-            options = options || {};
+        const body = composeSenderRequestBoddyForOneMessage(message, options);
+        return makeRequest(client, body);
+    };
+}
+
+/**
+    The sendAll Method for multiple messages
+    @param {Array}  messages - An array of objects containing the fields described in send Method [{message, options}, ... ]
+*/
+function sendAll(client) {
+    return function sendAll(messages) {
+        const body = messages.map(x => composeSenderRequestBoddyForOneMessage(x.message, x.options));
```

Comment:
X here is just a placeholder for the arrow function. It could be `m` that sounds more like "message" but in the end it's just a variable like in `for ( int i = 0...` or `f(x)`, meaning each item from the array.